targettests/execution/auto_kernel.cpp

```diff
@@ -6,10 +6,7 @@
  * the terms of the Apache License 2.0 which accompanies this distribution.    *
  ******************************************************************************/
 
-// RUN: nvq++ -std=c++17 --enable-mlir %s -o %t && %t | FileCheck %s
-// RUN: if [ $(echo %cpp_std | cut -c4- ) -ge 20 ]; then \
```

Comment:
This line aborts when %cpp_std is not defined.

Reply:
> This line aborts when %cpp_std is not defined.

Isn't this line in many of our tests? What's special about this one?

Reply:
ech, bash is not very helpful - I think the command exit code is 0 if the value is  not an integer.
This should work as intended:
`if [ -n  "$(echo %cpp_std)" ] && [ $(echo %cpp_std | cut -c9- ) -ge 20 ]; then`